pxr/imaging/plugin/hdRpr/python/generateRenderSettingFiles.py

```diff
@@ -0,0 +1,530 @@
+import sys, os, argparse
+
+dry_run = False
+
+render_setting_categories = [
+    {
+        'name': 'Denoise',
+        'settings': [
+            {
+                'name': 'enableDenoising',
+                'ui_name': 'Enable Denoising',
+                'defaultValue': False,
+                'custom_houdini_tags': [
+                    '"uiicon" VIEW_display_denoise'
+                ]
+            }
+        ]
+    },
+    {
+        'name': 'Sampling',
+        'settings': [
+            {
+                'name': 'maxSamples',
+                'ui_name': 'Max Pixel Samples',
+                'help': 'Maximum number of samples to render for each pixel.',
+                'defaultValue': 256,
+                'minValue': 1,
+                'maxValue': 2 ** 16
+            }
+        ]
+    },
+    {
+        'name': 'AdaptiveSampling',
+        'settings': [
+            {
+                'name': 'minAdaptiveSamples',
+                'ui_name': 'Min Pixel Samples',
+                'help': 'Minimum number of samples to render for each pixel. After this, adaptive sampling will stop sampling pixels where noise is less than \'Variance Threshold\'.',
+                'defaultValue': 64,
+                'minValue': 1,
+                'maxValue': 2 ** 16
+            },
+            {
+                'name': 'varianceThreshold',
+                'ui_name': 'Variance Threshold',
+                'help': 'Cutoff for adaptive sampling. Once pixels are below this amount of noise, no more samples are added. Set to 0 for no cutoff.',
+                'defaultValue': 0.0,
+                'minValue': 0.0,
+                'maxValue': 1.0
+            }
+        ]
+    },
+    {
+        'name': 'Quality',
+        'settings': [
+            {
+                'name': 'maxRayDepth',
+                'ui_name': 'Max Ray Depth',
+                'help': 'The number of times that a ray bounces off various surfaces before being terminated.',
+                'defaultValue': 8,
+                'minValue': 2,
+                'maxValue': 50
+            },
+            {
+                'name': 'maxRayDepthDiffuse',
+                'ui_name': 'Diffuse Ray Depth',
+                'help': 'The maximum number of times that a light ray can be bounced off diffuse surfaces.',
+                'defaultValue': 3,
+                'minValue': 2,
```

Comment:
I think it's valid to have a min value of 1

Reply:
easy, I just took values from Maya plugin